src/logsearch-config/src/logstash-filters/snippets/app-logmessage-rtr.conf

```diff
@@ -59,12 +59,15 @@ if ( [@type] == "LogMessage" and [@source][type] == "RTR" ) {
           replace => {"@message" => "%{[rtr][status]} %{[rtr][verb]} %{[rtr][path]} (%{[rtr][response_time_ms]} ms)"}
         }
 
-
         # Set @level (based on HTTP status)
-        if [rtr][status] >= 400 {
+        if [rtr][status] >= 500 {
```

Comment:
This might be really useful for metrics and dashboards later. @jontours we should look at this at some point.